### ISSUE TITLE
AsyncWorldEdit Support?

### ISSUE THREAD

**Issue by ColorsWind** [rlf/uSkyBlock#1228]
_Please paste the output from `/usb version` below_
[14:17:31 INFO]: Name: uSkyBlock
[14:17:31 INFO]: Version: 2.8.3
[14:17:31 INFO]: Description: Ultimate SkyBlock v2.8.3-685d633
[14:17:31 INFO]: Language: cn (cn)
[14:17:31 INFO]:   State: d=256, r=128, i=90, p=104, n=false, awe=false
[14:17:31 INFO]: Server: Paper git-Paper-183 (MC: 1.14.4)
[14:17:31 INFO]:   State: online=false, bungee=true
[14:17:31 INFO]: ------------------------------
[14:17:31 INFO]: Vault 1.7.2-b107 (ENABLED)
[14:17:31 INFO]: WorldEdit 7.0.1;61bc012 (ENABLED)
[14:17:31 INFO]: WorldGuard 7.0.1-SNAPSHOT;556b638 (ENABLED)
[14:17:31 INFO]: ------------------------------
paste it here please (replace this text)
```

_What steps will reproduce the problem?_
1. Install AsyncWorldEdit
2. Create an islands with more than 500,000 blocks
3. It will take about 8 sec
4. Uninstall AsyncWorldEdit. Repeat the above steps.
5. It will take almost the same time

_If you have any log-files, please paste them to [pastebin.com](http://pastebin.com)_
https://pastebin.com/4SqHy4CS
https://pastebin.com/ZPcVuEVE

**Comment by Muspah** [comment]
Hi,

We currently don't offer specific AWE support. We used to support FAWE, but there is no official FAWE 1.14 build available. I've looked into supporting AWE, but I can't say if and when it will become available for now.

**Comment by Muspah** [comment]
Fixed in #1264
